docs/usage/nfd-master.md

```diff
@@ -29,6 +29,11 @@ are created on the node (note the allowed
 > its [`-enable-nodefeature-api`](../reference/worker-commandline-reference.md#-enable-nodefeature-api)
 > flag.
 
+When `-enable-nodefeature-api` option is enabled and NFD-Master is intended to run
+with more than one replica, it is advised to use `-enable-leader-election` flag.
+This flag turns on leader election for NFD-Master and let only one replica
+to update `NodeFeature` objects.
```

Comment:
nfd-master only updates Nodes, it just "listens" to NodeFeature and NodeFeatureRule objects (as we don't have e.g. status field in those, yet).

```suggestion
to act on changes in NodeFeature and NodeFeatureRule objects.
```

Reply:
:+1: